packages/next/src/client/components/forbidden.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  HTTP_ERROR_FALLBACK_ERROR_CODE,
+  type HTTPAccessFallbackError,
+} from './http-access-fallback/http-access-fallback'
+
+// TODO: Add `forbidden` docs
+/**
+ * @experimental
+ * This function allows you to render the [not-found.js file](https://nextjs.org/docs/app/api-reference/file-conventions/not-found)
```

Comment:
comment needs updating. probably need to check the comment for unauthroized one too